src/index.js

```diff
@@ -28,6 +28,10 @@ const requestPromised = promisify(request, Promise);
 
 function createSauceConnectProcess (options) {
     return new Promise((resolve, reject) => {
+        if (options.useExistingTunnel){
+            resolve();

```

Comment:
It can lead to an unhandled exception on disconnecting because disconnecting code assumes that `sauceConnectProcess` always has a value. We should add a check for it to the `disconnect` method. And let's move the check for `useExistingTunnel` to the `connect` method for symmetry. If you refactor it this way, please don't forget to remove this flag from `sauceConnectOptions` at [line 70](https://github.com/AlexanderMoskovkin/saucelabs-connector/pull/27/files#diff-1fdf421c05c1140f6d71444ea2b27638R70).